packages/playwright-core/src/server/bidi/bidiPage.ts

```diff
@@ -61,14 +60,15 @@ export class BidiPage implements PageDelegate {
     this._realmToContext = new Map();
     this._page = new Page(this, browserContext);
     this._browserContext = browserContext;
-    this._networkManager = new BidiNetworkManager(this._session, this._page, this._onNavigationResponseStarted.bind(this));
+    this._networkManager = new BidiNetworkManager(this._session, this._page, () => {});
```

Comment:
Let's remove `_onNavigationResponseStarted` from `_onNavigationResponseStarted` as it is not used anymore.

Reply:
I assume you meant "remove `_onNavigationResponseStarted` from `BidiNetworkManager`". Done.